configuration.go

```diff
@@ -116,20 +117,13 @@ func (v *VirtualMachineConfiguration) SetNetworkDevicesVirtualMachineConfigurati
 	}
 	array := objc.ConvertToNSMutableArray(ptrs)
 	C.setNetworkDevicesVZVirtualMachineConfiguration(objc.Ptr(v), objc.Ptr(array))
+	v.networkDeviceConfiguration = cs
 }
 
 // NetworkDevices return the list of network device configuration set in this virtual machine configuration.
 // Return an empty array if no network device configuration is set.
 func (v *VirtualMachineConfiguration) NetworkDevices() []*VirtioNetworkDeviceConfiguration {
-	nsArray := objc.NewNSArray(
-		C.networkDevicesVZVirtualMachineConfiguration(objc.Ptr(v)),
-	)
-	ptrs := nsArray.ToPointerSlice()
-	networkDevices := make([]*VirtioNetworkDeviceConfiguration, len(ptrs))
-	for i, ptr := range ptrs {
-		networkDevices[i] = newVirtioNetworkDeviceConfiguration(ptr)
-	}
-	return networkDevices
+	return v.networkDeviceConfiguration
```

Comment:
I decided to take this concise approach because it is difficult to identify the interface NetworkDeviceAttachment class in the Objective-C world and restore it in the Go world.

This also has the advantage that it is identical to the VirtioNetworkDeviceConfiguration pointer at set time.